go.mod

```diff
@@ -145,5 +145,6 @@ require (
 // use cosmos-compatible protobufs
 replace (
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20200815110645-5c35d600f0ca // indirect
```

Comment:
what's the reason we needed bump go level db version?

Reply:
There was an error:
`
tokenfactory % bash dockernet/tests/run_all_tests.sh 
Error: rpc error: code = InvalidArgument desc = failed to load state at height 6; version does not exist (latest height: 6): invalid request
`
So we bump to goleveldb version osmosis is using.

Reply:
> There was an error when we try query: 

`tokenfactory % bash dockernet/tests/run_all_tests.sh Error: rpc error: code = InvalidArgument desc = failed to load state at height 6; version does not exist (latest height: 6): invalid request` 

So we bump to goleveldb version osmosis is using.